ui/desktop/app/controllers/scopes/scope/projects/targets/index.js

```diff
@@ -174,4 +183,41 @@ export default class ScopesScopeProjectsTargetsIndexController extends Controlle
   applyFilter(selectedItems) {
     this.scopes = [...selectedItems];
   }
+
+  @action
+  toggleSessionsFlyout() {
+    this.sessionsFlyoutActive = !this.sessionsFlyoutActive;
```

Comment:
Do you need to set this boolean? Couldn't you just be checking if you have a `selectedTarget`? And this method then wouldn't need to exist since on close you could just use `selectTarget` and set the target to null

Reply:
yes, good point, ill clean this up.